shinylive_lock.json

```diff
@@ -97,7 +97,7 @@
   },
   "plotnine": {
     "name": "plotnine",
-    "version": "0.0.post20+g64295e1",
+    "version": "0.0.post1930+g64295e1",
```

Comment:
I think this change shouldn't be committed -- I believe your local copy of plotnine is getting `post1930` probably because it doesn't have the tags cloned, or something like that. (This is another case where the auto-numbering based on commits since the last tag is difficult to deal with.)

Reply:
oh shoot, yeah I meant to do a `git add --patch` here but got carried away when adding the file to the commit, I'll fix it.

Reply:
Done in 2c4e1a26a20ca6d94522a5badb57dcbf7a731bba